src/Build/Collections/ItemDictionary.cs

```diff
@@ -227,11 +231,19 @@ public ICollection<T> GetItems(string itemType)
         /// <summary>
         /// Whether the provided item is in this table or not.
         /// </summary>
-        public bool Contains(T projectItem)
+        bool ICollection<T>.Contains(T projectItem)
```

Comment:
Hid this since it isn't actually used after this change, and only exists to fulfil `ICollection<T>` so that `CopyOnReadEnumerable` can avoid a list allocation.